docs/reference/module-api/module-overview.md

```diff
@@ -1,6 +1,9 @@
 # Module properties
 
-Module configuration objects may use the following configuration properties.
+Module configuration objects may use the following configuration properties. The overall categories are broadly defined:
+- [Configuration settings](#configuration-settings): Static module settings. Once the module is initialized these settings are fixed and can't access the module itself or any other module's settings.
```

Comment:
```suggestion
- [Configuration settings](#configuration-settings): Static module settings. Once the module is initialized, these settings become fixed, and can't access the module itself or any other module's settings.
```

 I had this in my suggestion but it was illegally done.

Reply:
This one doesn't seem right to me. They don't become fixed after initialization. After initialization they are fixed.